input-spec.json

```diff
@@ -2231,36 +2287,90 @@
         "pointer": "/boundary_conditions/neumann_boundary/*",
         "type": "object",
         "default": null,
-        "optional": [
+        "required": [
```

Comment:
Made these required fields in the NBC. I am not sure why it had a default ID of `1` and value of `[0, 0, 0]`